server/rpc.go

```diff
@@ -243,22 +243,22 @@ func genOrchestratorReq(b common.Broadcaster) (*net.OrchestratorRequest, error)
 }
 
 func getOrchestrator(orch Orchestrator, req *net.OrchestratorRequest) (*net.OrchestratorInfo, error) {
-	addr := ethcommon.BytesToAddress(req.Address)
-	if err := verifyOrchestratorReq(orch, addr, req.Sig); err != nil {
+	bAddr := ethcommon.BytesToAddress(req.Address)
+	if err := verifyOrchestratorReq(orch, bAddr, req.Sig); err != nil {
 		return nil, fmt.Errorf("Invalid orchestrator request (%v)", err)
 	}
 
 	// currently, orchestrator == transcoder
-	return orchestratorInfo(orch, addr, orch.ServiceURI().String())
+	return orchestratorInfo(orch, bAddr, orch.Address(), orch.ServiceURI().String())
 }
 
-func orchestratorInfo(orch Orchestrator, addr ethcommon.Address, serviceURI string) (*net.OrchestratorInfo, error) {
-	priceInfo, err := orch.PriceInfo(addr)
+func orchestratorInfo(orch Orchestrator, bAddr ethcommon.Address, oAddr ethcommon.Address, serviceURI string) (*net.OrchestratorInfo, error) {
```

Comment:
I ended up reverting back to the original `orchestratorInfo()` signature in 0d73205 because the orch address can be accessed via the `orch Orchestrator` parameter that is already passed into the function so we don't have to worry about additional parameter naming right now. At some point if the `OrchestratorInfo.Address` field needs to be populated with a value that is not accessible via `orch Orchestrator` then we can consider updates to the `orchestratorInfo()` signature then.